circuits/storage/src/lib.rs

```diff
@@ -0,0 +1,143 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use cargo_toml::{Dependency, Manifest};
+use dusk_plonk::prelude::{Circuit, Compiler, PublicParameters};
+use once_cell::sync::Lazy;
+use rand::rngs::StdRng;
+use rand::SeedableRng;
+use rusk_profile::{Circuit as CircuitProfile, Theme};
+use std::env;
+use std::io::{Error as ioError, ErrorKind, Result as ioResult};
```

Comment:
You should probably import them like this:

```suggestion
use std::io::{Error as IoError, ErrorKind, Result as IoResult};
```

Or even better let the namespace do the work:

```suggestion
use std::io;
```

and use 

```rust
fn foo() -> io::Result<()> {
    Ok(())
}
```

I'm surprised clippy doesn't complain about lowercase imports.

Reply:
yeah, I might have overcompensated. I'll use `io::Error` and `io::Result` but will keep `ErrorKind`